.ci.yaml

```diff
@@ -177,6 +177,13 @@ targets:
       - os=Linux
     dimensions:
       kvm: "1"
+    runIf:
+      - DEPS
+      - .ci.yaml
+      - tools/clang_tidy/**
+      - ci/builders/**
+      - ci/clang_tidy.sh
+      - shell/platform/fuchsia/**
     # TODO(https://github.com/flutter/flutter/issues/138559): Re-enable/delete.
     # runIfNot:
```

Comment:
I didn't remove the commented out `runIfNot` code, since https://github.com/flutter/flutter/issues/138559 is being investigated.  I can do so though if @matanlurey prefers.